pkg/remoteconfig/updater.go

```diff
@@ -59,11 +59,12 @@ type RcServiceConfiguration struct {
 
 // DatadogAgentRemoteConfig contains the struct used to update DatadogAgent object from RemoteConfig
 type DatadogAgentRemoteConfig struct {
-	ID            string                       `json:"id,omitempty"`
-	Name          string                       `json:"name,omitempty"`
-	CoreAgent     *CoreAgentFeaturesConfig     `json:"config,omitempty"`
-	SystemProbe   *SystemProbeFeaturesConfig   `json:"system_probe,omitempty"`
-	SecurityAgent *SecurityAgentFeaturesConfig `json:"security_agent,omitempty"`
+	ID            string                        `json:"id,omitempty"`
+	Name          string                        `json:"name,omitempty"`
+	CoreAgent     *CoreAgentFeaturesConfig      `json:"config,omitempty"`
+	SystemProbe   *SystemProbeFeaturesConfig    `json:"system_probe,omitempty"`
+	SecurityAgent *SecurityAgentFeaturesConfig  `json:"security_agent,omitempty"`
+	CRDs          *CustomResourceDefinitionURLs `json:"crds,omitempty"`
```

Comment:
Take this with a grain of salt since I'm not too familiar with the operator, but this looks to me like structurally, configs are broken out by agent binary. Should the CRD configs be wrapped inside a ClusterAgentFeatureConfig type?

Reply:
Yeah that's a great call. I think this is a good idea for future work

Reply:
[51d2cf9](https://github.com/DataDog/datadog-operator/pull/1340/commits/51d2cf97370b4a5278df24135b119e4816e96415)